database/frontend/react.json

```
@@ -114,5 +114,12 @@
     "url": "https://www.theodinproject.com/",
     "category": "frontend",
     "subcategory": "react"
+  },
+  {
+    "name": "Redux Toolkit Tutorial",
+    "description": "In this tutorial series on Redux Toolkit for beginners, we dive into the what, why and the how.",

```

[proofreading] just a typo
```suggestion
    "description": "In this beginner-level tutorial series on Redux Toolkit, we dive into the what, why, and the how.",

```

---

Done!